kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java

```diff
@@ -1595,4 +1634,32 @@ private WifiMode getWifiConfigWirelessMode(String mode) {
         }
         return wifiMode;
     }
+
+    private CellularModem getModemService(String ifaceName, ModemManagerService mms) {
+        CellularModem cellularModem = mms.getModemService(ifaceName);
+        if (cellularModem != null) {
+            return cellularModem;
+        }
+        Collection<CellularModem> modemServices = mms.getAllModemServices();
+        for (CellularModem modem : modemServices) {
```

Comment:
Isn't anything missing in this loop?

Reply:
Yes, need to break out of the loop if not null. Thank you!